packages/wrangler/src/create-worker-preview.ts

```diff
@@ -139,19 +139,18 @@ async function createPreviewToken(
 
   return {
     value: preview_token,
-    host: ctx.zone
-      ? ctx.zone.host
-      : worker.name
-      ? `${
-          worker.name
-          // TODO: this should also probably have the env prefix
-          // but it doesn't appear to work yet, instead giving us the
-          // "There is nothing here yet" screen
-          // ctx.env && !ctx.legacyEnv
-          //   ? `${ctx.env}.${worker.name}`
-          //   : worker.name
-        }.${host.split(".").slice(1).join(".")}`
-      : host,
+    host:
+      ctx.host ?? worker.name
+        ? `${
```

Comment:
this would not use ctx.host if it's available no? as in, if ctx.host and worker.name are both available, then it'll use worker.name, which isn't the desired behaviour I think?